usaspending_api/agency/v2/views/object_class_count.py

```diff
@@ -29,16 +29,21 @@ def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         )
 
     def get_object_class_count(self):
+        q = (
```

Comment:
same thing as before but with the /count/ endpoints. looks like both /object_class/count/ and program_activity/count/ do it.